packages/mui-material/src/OutlinedInput/NotchedOutline.test.js

```diff
@@ -54,4 +54,11 @@ describe('<NotchedOutline />', () => {
       paddingRight: '8px',
     });
   });
+  it('should not set padding (notch) for empty, null or undefined label props', () => {
+    const spanStyle = { paddingLeft: '', paddingRight: '' };
```

Comment:
```suggestion
    const spanStyle = { paddingLeft: '0px', paddingRight: '0px' };
```

Looks like test are failing on this one - https://app.circleci.com/pipelines/github/mui-org/material-ui/61625/workflows/2c31ec9f-95e7-40f3-9bdb-cc0ca3797af8/jobs/335999